libsolidity/experimental/codegen/IRGeneratorForStatements.cpp

```diff
@@ -321,14 +341,13 @@ void IRGeneratorForStatements::endVisit(FunctionCall const& _functionCall)
 	// TODO: get around resolveRecursive by passing the environment further down?
 	functionType = m_context.env->resolveRecursive(functionType);
 	m_context.enqueueFunctionDefinition(functionDefinition, functionType);
-	// TODO: account for return stack size
-	m_code << "let " << IRNames::localVariable(_functionCall) << " := " << IRNames::function(*m_context.env, *functionDefinition, functionType) << "(";
+	m_code << "let " << var(_functionCall).commaSeparatedList() << " := " << IRNames::function(*m_context.env, *functionDefinition, functionType) << "(";
```

Comment:
You'll need to account for the case that the stack size of the function return is zero, in which case ``var(_functionCall).commaSeparatedList()`` is empty and you can't declare a variable (see https://github.com/ethereum/solidity/pull/14660#discussion_r1377569051 only that the condition here should not be "no return parameters", but rather "return parameters of stack size zero")

Reply:
Is this true to any variable with stack size zero or only for the function call case (and of course binary operation)?